native/src/main/java/io/ballerinax/salesforce/Utils.java

```diff
@@ -0,0 +1,31 @@
+package io.ballerinax.salesforce;
```

Comment:
License headers?

Reply:
Updated in [Update license headers](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/21a0879735857d7edec64e1999da5151ad54b223)